packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -192,7 +192,7 @@ export default class VList {
         let lastList: Node;
 
         // Use a placeholder to hold the position since the root list may be moved into document fragment later
-        this.rootList.parentNode!.replaceChild(placeholder, this.rootList);
+        this.rootList.parentNode?.replaceChild(placeholder, this.rootList);
```

Comment:
This means if parentNode is null, replaceChild won't happen.

Do you know in what case this can happen? We'd better understand why it can happen but not just do null check, because there might be underlying issue causes the issue so here the null issue is just a symptom of the real issue. If that is the case, checking null here will hide the real issue.